Presentation/WindowsClient/Controls/Importer.cs

```diff
@@ -156,9 +154,8 @@ public async Task CleanTable(DataTable table)
             table.RemoveDuplicateRows();
 
             var type = await InvokeQuery(new EntityTypeQuery() { Name = table.TableName });
```

Comment:
good idea not to use type as a variable name

Reply:
Even when the object is an instance of `Type`? Because if we want to make those changes there are a few places in the code I'll need to fix that.

Reply:
I could make it more explicit, such as
```suggestion
            Type type = await InvokeQuery(new EntityTypeQuery() { Name = table.TableName });
```
or would you still prefer something like
```suggestion
            Type entityType = await InvokeQuery(new EntityTypeQuery() { Name = table.TableName });
```

Reply:
var entityType is probably sufficient. 
type is too generic ie: could be string, int, Table so it's a bad variable name.